### DISCUSSION TITLE
Configuring Notification using Cortex Alertmanager

### DISCUSSION THREAD

**Discussion by aurang-zeb313** [cortexproject/cortex#5999] [Q&A]
Hi everyone,

I have an assignment of configuring notification in cortex ruler and when notification fired i have to consume it and used in my api.
can some one provide clear documentation  with examples.
1. How to download, install and configure cortex  alert manager for cortex version v1.14.0
2. How to integrate ruler and alert manager 
3. How to consume alert manager in my api
Please some one help by providing a clear documentation  along with useful examples.

**Comment by friedrichg** [comment]
1. Cortex alertmanager is included in every cortex version. You just need to pass the proper `-target` to it. In this case 'alertmanager'. But you should not use cortex v1.14.0, that is a very old version. Try latest

2. You have a few options
- https://cortexmetrics.io/docs/getting-started/#running-cortex-as-a-single-instance
- https://cortexmetrics.io/docs/getting-started/#running-cortex-in-microservice-mode
- Just deploy cortex helm chart in any cluster you have 
These two include ruler and alertmanager. Try them out and let me know 

3. Alertmanager API original API is on https://prometheus.io/docs/alerting/latest/alertmanager/. But I think I can we can do better to document the process. Here is the issue https://github.com/cortexproject/cortex/issues/6000
